src/content-handlers/iiif/modules/uv-mediaelementcenterpanel-module/js/source-chooser-fixed.js

```diff
@@ -0,0 +1,301 @@
+/**
+ * https://github.com/mediaelement/mediaelement-plugins/blob/master/dist/source-chooser/source-chooser.js
+ * Version: v2.5.1
+ * Last Copied: 2024-10-16
+ * License: MIT (https://johndyer.mit-license.org)
+ *
+ * This was done to:
+ * 1. fix the improper use of .siblings() on line 144 (of the original).
+ * 2. improve keyboard support of the source chooser button and menu (line 67).
+ *
+ * This will be contributed upstream when fixed and stable.
+ * Chris Hallberg (challber@villanova.edu)
```

Comment:
Maybe this comment should be replaced with a link to the upstream pull request, if we're contemplating merging this as-is due to lack of upstream activity. That way, if in future we try to get rid of this file, we understand why it's here and how to know when it's no longer needed.